src/Coordination/Changelog.h

```diff
@@ -23,9 +23,10 @@ using IndexToLogEntry = std::unordered_map<uint64_t, LogEntryPtr>;
 enum class ChangelogVersion : uint8_t
 {
     V0 = 0,
+    V1 = 1, /// with 64 bit buffer header
```

Comment:
It is not in production, why even introduce a new version?

Reply:
Actually it is Ok to just test the versioning right now.
We can drop support for old versions later.

Reply:
mmm, why not?) `1` Looks better than `0`)